docs/source/admin/fs/local-fs.rst

```diff
@@ -762,3 +764,37 @@ If you want FSCrawler to follow the symbolic links, you need to be explicit abou
    name: "test"
    fs:
      follow_symlink: true
+
+Tika Config Path
+^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.10
+
+If you want to override the default tika parser configuration, you can set the path to a custom tika
+configuration file, which will be used instead.
+
+.. code:: yaml
+
+   name: "test"
+   fs:
+     tika_config_path: '/path/to/tikaConfig.xml'
+
+An example tika config file is shown below. See `apache documentation <https://tika.apache.org/1.27/configuring.html>`__ for more information.
```

Comment:
We should make this link dynamic.

Could you add the following lines in `docs/source/conf.py`:

```python
.. |Tika_configuring| replace:: Configuring Tika
```

just after:

```python
.. |Tika_format| replace:: Tika
.. |Tika_version| replace:: Tika {fmt_tika_version}
```

and

```python
.. _Tika_configuring: https://tika.apache.org/{fmt_tika_version}/configuring.html
```

just after:

```python
.. _Tika_format: https://tika.apache.org/{fmt_tika_version}/formats.html#Supported_Document_Formats
.. _Tika_version: https://tika.apache.org/{fmt_tika_version}/
```

Then replace here:

```rst
See `apache documentation <https://tika.apache.org/1.27/configuring.html>`__ for more information.
```

with

```rst
See |Tika_configuring|_ for more information.
```